controller/space_codebases.go

```diff
@@ -56,6 +56,7 @@ func (c *SpaceCodebasesController) Create(ctx *app.CreateSpaceCodebasesContext)
 			SpaceID: spaceID,
 			Type:    *reqIter.Attributes.Type,
 			URL:     *reqIter.Attributes.URL,
+			//TODO: We don't have the StackID here.
```

Comment:
I guess this will be filled whenever we expose the stack id to users. Should I keep as it is ?